### ISSUE TITLE
color contrast and visual feedback when reaching targets

### ISSUE THREAD

**Issue by lkeegan** [ssciwr/vstt#263]
Some of the older people had some difficulties seeing the circles and recognizing if they correctly reached them. We, therefore, wondered whether it would be possible:
- Adding a thin e.g. black line/circle around the targets and center to create a better color contrast (or/and making the gray color of the targets/center a bit lighter).
- Adding an option that can be selected in the settings and causes the target and center to turn the color (e.g. green) when the participant correctly reaches the target or center. This means the target or center that need to be reached turns from red to e.g. green when the cursor hits it. The green color could appear for a given time (e.g. 160 ms) before the circle color turns back into the standard gray. This should just signal that the participant reached the correct target in time.
- Adding an option that can be selected in the settings that causes the circle (target and center) to turn e.g. white if the cursor is inside it. This means if a person moves the cursor inside a target or the center (e.g. before it is displayed in red or even if the cursor is in the wrong target) it turns from gray to e.g. white. This is simply to signal that the cursor is inside the circle.

**Comment by lkeegan** [comment]
As discussed with @ZoeJacky the initial plan is to make the target go from red to green when the cursor hits it, and it would then stay green until the next target becomes red, rather than for a fixed amount of time.
This is simpler to implement, and also avoids having multiple coloured targets displayed at the same time, which may be confusing.

A possible implementation could be to add a new variable e.g. `green_target_index=None` here: https://github.com/ssciwr/vstt/blob/d866c0832ee7c4064ac6c0801cf4b61daf41ce50/src/vstt/task.py#L289

Then in the loop below make the target with index `green_target_index` green.

At the end of the loop where success is determined, `green_target_index` can be updated accordingly:
https://github.com/ssciwr/vstt/blob/d866c0832ee7c4064ac6c0801cf4b61daf41ce50/src/vstt/task.py#L407-L410

**Comment by lkeegan** [comment]
resolved by #279
